layers/+lang/c-c++/funcs.el

```diff
@@ -393,10 +396,12 @@
 (defun spacemacs//c-c++-organize-includes-on-save ()
   "Organize the includes on save when `c-c++-enable-organize-includes-on-save'
```

Comment:
This comment is now wrong, given that this function now works without conditions.